packages/core/src/Stores/client-store.js

```diff
@@ -462,7 +462,7 @@ export default class ClientStore extends BaseStore {
                     this.is_passkey_supported &&
                     this.root_store.ui?.is_mobile
```

Comment:
can you double check if ui.ismobile updated with new breakpoints with tablet?

Reply:
yes, isMobile in ui store has the same breakpoint as for useDevice from deriv-ui.
the notification is not visible on tablet, @amina-deriv